src/app/(auth)/createprofile.tsx

```diff
@@ -1,16 +1,86 @@
 import { Text, View } from "@/src/components/Themed";
 import { resetNavigationStack } from "@/src/lib/resetNavigationStack";
-import { TouchableOpacity } from "react-native";
+import { Image, TextInput, TouchableOpacity } from "react-native";
+import DefaultColors from "@/src/constants/DefaultColors";
+import { useColorScheme } from "nativewind";
+import { useEffect, useState } from "react";
+import { IconCirclePlus } from "@tabler/icons-react-native";
+import Icon from "@/src/components/Icon";
+import { fetchSingleUserThumbnail } from "@/src/lib/getRandomProfilePics";
 
 export default function Createprofile() {
+  const { colorScheme } = useColorScheme();
+  const [displayName, setDisplayName] = useState("");
+  const [username, setUsername] = useState("");
+  const [pfp, setPfp] = useState();
+
+  useEffect(() => {
+    fetchSingleUserThumbnail().then(user => {
+      setPfp(user.imgurl);
+    });
+  }, []);
+
   return (
-    <View className="mt-16">
-      <Text>Create Profile</Text>
+    <View className="flex-1 flex flex-col justify-between items-center pt-40 pb-20">
+      {/* Form */}
+      <View className="flex flex-col self-start gap-y-4 w-screen pl-5 pr-5">
+        <Text className="text-4xl font-bold self-center">Create account</Text>
+        <View className="flex flex-col gap-y-4">
+          {/* Profile Picture */}
+          <View className="flex flex-col gap-y-1">
+            <Text className="text-xs font-bold">Profile picture</Text>
+            <TouchableOpacity className="relative self-center"
+              onPress={() => {
+                // Logic for picking picture
+              }}>
+              {pfp && <Image className="h-24 w-24 rounded-3xl overflow-visible" source={{ uri: pfp }} />}
```

Comment:
Can you make it `rounded-full` so that it's a circle? I think that's better since pfps are circles everywhere else (just updated the design).

Also `overflow-visible` makes it so that the rounding doesn't show up, but when I remove that then it works. Was there some reason you added that?

Reply:
Actually I think the rounded square is good. You should be able to just use the big profile picture component now. But still deal with the overflow

Reply:
`overflow-visible` was when I was trying to put the plus icon over the profile picture, but yeah it works without it.